python/pyspark/pandas/tests/test_stats.py

```diff
@@ -283,7 +283,7 @@ def test_cov_corr_meta(self):
                 index=pd.Index([1, 2, 3], name="myindex"),
             )
             psdf = ps.from_pandas(pdf)
-            self.assert_eq(psdf.corr(), pdf.corr())
+            self.assert_eq(psdf.corr(), pdf.corr(), check_exact=False)
```

Comment:
Looks like it was mistakenly cherry-picked together.

Reply:
Thanks for replay